solr/bin/solr.cmd

```diff
@@ -292,7 +292,6 @@ IF "%FIRST_ARG%"=="-help" goto run_solrcli
 IF "%FIRST_ARG%"=="-usage" goto run_solrcli
 IF "%FIRST_ARG%"=="-h" goto run_solrcli
 IF "%FIRST_ARG%"=="--help" goto run_solrcli
-IF "%FIRST_ARG%"=="-help" goto run_solrcli
```

Comment:
`-help` is a special case (and we shoild probably comment that it is so) in the idea that folks might do -h, --help, and -help equally when trying to get to the help!

Reply:
Line 295 is a duplication of 291, which is why I removed it. Should probably not have an effect?